src/Components/Classes/index.js

```diff
@@ -1,11 +1,102 @@
+import { useEffect, useState } from 'react';
 import styles from './classes.module.css';
+import Form from './Form/Form';
+import Table from './Table/Table';
+
+const Classes = () => {
+  const [classes, setClasses] = useState([]);
+  const [activities, setActivities] = useState([]);
+  const [trainers, setTrainers] = useState([]);
+
+  const getClasses = async () => {
+    const response = await fetch(`${process.env.REACT_APP_API_URL}/api/classes/`);
+    const data = await response.json();
+    setClasses(data.data);
```

Comment:
If you make data as an objet it isn't necesary to use data.data

Reply:
Done